fdbserver/workloads/ThroughputQuota.actor.cpp

```diff
@@ -23,42 +23,46 @@
 
 #include "flow/actorcompiler.h" // This must be the last #include.
 
-class GlobalTagThrottlingWorkload : public TestWorkload {
+// This workload sets the throughput quota of a tag during the setup phase
+class ThroughputQuotaWorkload : public TestWorkload {
```

Comment:
It looks like this workload only tests that we're successfully able to set the quota for a tag. Did I understand that correctly? If so, is there a plan to test that the throttling works as expected and that transactions get rejected after `PROXY_MAX_TAG_THROTTLE` time?